lms/views/admin/application_instance.py

```diff
@@ -302,6 +302,7 @@ def update_instance(self):
             ("desire2learn", "client_id", str),
             ("desire2learn", "client_secret", aes_secret),
             ("desire2learn", "groups_enabled", bool),
+            ("desire2learn", "create_line_item", bool),
```

Comment:
This would be a sweet thing to somehow bundle up into the plugins. But this would require some way of abstracting the form you'd like to see to be useful. Form abstractions inevitably lead to madness, so maybe that's not a good idea?

In any case, not an idea for now.